package.json

```diff
@@ -16,31 +17,33 @@
     "@types/node": "^20.12.7",
     "@types/react": "^18.2.79",
     "@types/react-dom": "^18.2.25",
+    "@vitejs/plugin-react": "^4.3.1",
     "classnames": "^2.5.1",
     "localforage": "^1.10.0",
     "lodash": "^4.17.21",
     "match-sorter": "^6.3.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-error-boundary": "^4.0.12",
-    "react-scripts": "5.0.1",
     "sort-by": "^1.2.0",
     "typescript": "^5.4.5",
+    "vite": "^5.3.1",
+    "vite-tsconfig-paths": "^4.3.2",
     "web-vitals": "^2.1.4",
     "wouter": "^3.1.2"
   },
   "scripts": {
     "install-deps": "yarn --force",
     "typecheck": "tsc -p tsconfig.json --noEmit",
-    "start": "rimraf ./node_modules/.cache && NODE_OPTIONS=--max_old_space_size=8192 PORT=3004 BROWSER=none react-scripts start",
-    "start:https": "HTTPS=true yarn start",
-    "build": "react-scripts build",
+    "dev": "NODE_OPTIONS=--max_old_space_size=8192 BROWSER=none vite",
```

Comment:
could you test this on your end as well? I've tried removing the old `NODE_OPTIONS` hack and it seems to work now!

Reply:
nice, I tested it locally and it works without the memory increase now!